gallery/plot_visualization_utils.py

```diff
@@ -252,7 +237,7 @@ def show(imgs):
 # Instance segmentation models
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Instance segmentation models have a significantly different output from the
+# Instance segmentation models have a slightly different output from the
```

Comment:
I'm not sure why we said that MaskRCNN has a significantly different output than FasterRCNN here. It just includes the same as FasterRCNN + the mask info.

Reply:
The masks are quite different: one is a non-normalized score while for the other it's probabilities directly. Also one can detect an arbiratry number of classes while for the other one it's fixed, right?

(I'm fine with the change regardless)

Reply:
happy to revert